### DISCUSSION TITLE
Textadept 11.4

### DISCUSSION THREAD

**Discussion by orbitalquark** [orbitalquark/textadept#245] [General]
Textadept 11.4 is released from https://github.com/orbitalquark/textadept/releases/tag/textadept_11.4.

This is a major new release with the following notable changes from 11.3:

* GUI toolkit updated to Gtk 3 from Gtk 2. Look and feel will likely be different.
* 64-bit Windows executables; 32-bit exe support was dropped.
* macOS 10.10 (Yosemite) is the new minimum required version of macOS.

Please see the [changelog](https://orbitalquark.github.io/textadept/changelog.html) for a complete list of changes from 11.3.

Bugfixes since 11.4 beta:

* Fixed line comment toggling for Batch.
* Fixed lack of HTML documentation in releases since 11.3.
* Ensure the statusbar is updated after `buffer:set_lexer()`.
* Ensure `events.LEXER_LOADED` is emitted on `buffer.new()`.
* LSP: Fixed off-by-one errors for goto definition et. al. and find references.
* LSP: Fixed bug attempting to start a language server manually.
* Lua REPL: Fixed key bindings.
* Ruby: Fixed bug when trying to toggle a block at the end of the buffer.
* Scintilla: Fixed hiding selection when `view.selection_layer` is `view.LAYER_UNDER_TEXT`.
* Scintilla: Fix potential issues with drawing non-UTF-8 text.

Changes since 11.4 beta:

* Changed line duplication to line/selection duplication.
* Added [`ui.popup_menu()`][] for displaying menus like the right-click context menu.
* The GUI version now recognizes the 'menu' key.
* Added support for Hare.
* Updated Spanish translation.
* Updated R, Fortran, and Go lexers.
* Updated to Scintilla 5.2.4.

[`ui.popup_menu()`]: https://orbitalquark.github.io/textadept/api.html#ui.popup_menu

**Comment by gabdub** [comment]
The windows download link is broken (just remove 32 from the link).

**Comment by orbitalquark** [comment]
Thanks! Fixed.

**Reply by gabdub** [reply]
Also check github release note:
![imagen](https://user-images.githubusercontent.com/11639609/182158195-729fa075-e812-4776-94d1-64dd9e4f0714.png)

**Comment by orbitalquark** [comment]
Thanks again. Hopefully that's the last of them :)

**Comment by mhwombat** [comment]
NixOS users: With luck, the new version (11.4) of textadept should be available in unstable soon. I updated the derivation and submitted a [pull request](https://github.com/NixOS/nixpkgs/pull/184876). Here's the relevant [issue](https://github.com/NixOS/nixpkgs/issues/184865).
